### ISSUE TITLE
Crash when killing mobs with a saw

### ISSUE THREAD

**Issue by soadrage7654** [WesCook/Homestead#73]
I set up a mob grinder that funnels mobs into a Better With Mods saw, but the game crashes every time the saw kills a mob. It appears to be an incompatibility between the Better With Mods and Bagginses mods. Crash report attached.

[crash-2017-12-10_10.03.25-server.txt](https://github.com/WesCook/Homestead/files/1545792/crash-2017-12-10_10.03.25-server.txt)

**Comment by WesCook** [comment]
I'm able to confirm the crash.  Unfortunately I'm not sure how I can really address this one.  Neither mod is still offering support for 1.10.2.

It seems to be primarily on Bagginses side, based on the stacktrace.  Unfortunately Bagginses appears to be completely abandoned.  One of their last commits was a PR I made back in December of 2016, and I'm not sure that ever made it into a release version.

Basically the only way I can see to fix this is to fork Bagginses and release a distributed version.  The bug looks to be in asm code though, and that is way above my skill level.

I hate to do it, but I think I have to mark this one as `wontfix`.  I don't see any solution.

Thanks for the report though, and I'm sorry your farm won't work.

**Comment by soadrage7654** [comment]
Thanks for looking into it. I haven't logged into Bagginses much, but do you think there is a config option that might prevent this? Like "mobs don't drop bags on death"?

**Comment by WesCook** [comment]
I tested the only option that sounded relevant (`B:"Should bags be able to pick up items when dropped"=true`), but setting it to false had no effect.

**Comment by soadrage7654** [comment]
Alright, well thanks for looking into it.
